src/main/scala/chisel3/tester/package.scala

```diff
@@ -39,11 +40,19 @@ package object tester {
         require(x.binaryPoint == value.binaryPoint, "binary point mismatch")
         pokeBits(x, value.litValue)
       }
-      case (x: Bundle, value: Bundle) => {
+      case (xx: Bundle, value: Bundle) => {
         // TODO: chisel needs to expose typeEquivalent
-        require(x.elements.keys == value.elements.keys)  // TODO: this discards type data =(
-        (x.elements zip value.elements) foreach { case ((_, x), (_, value)) =>
-          x.poke(value)
+        require(xx.elements.keys == value.elements.keys)  // TODO: this discards type data =(
+        (xx.elements zip value.elements) foreach { case ((fieldName, xxx), (_, value)) =>
+          try {
```

Comment:
Not a fan of try/catch if alternatives are available. NoSuchElementException does not seem to be a stable API to rely on either. Can you do litOption or something instead?

Reply:
I was mostly trying to illustrate an improved error message. The message without this code `None.get` is pretty unhelpful and will be even more of a problem with real complex nested bundles like one sees in riscv-mini and rocket. Stability and robustness are important of course. I did not see an immediate way of getting a better access to the name of the Bundle involved.